src/Discord.Net.Core/Entities/ApplicationRoleConnection/RoleConnection.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+
+namespace Discord;
+
+/// <summary>
+///     Represents the connection object that the user has attached.
+/// </summary>
+public class RoleConnection
+{
+    /// <summary>
+    ///     Gets the vanity name of the platform a bot has connected.
+    /// </summary>
+    public string PlatformName { get; }
+
+    /// <summary>
+    ///     Gets the username on the platform a bot has connected.
```

Comment:
```suggestion
    ///     Gets the username on the platform a bot has connected to.
```

Reply:
not sure what to do here
this comment comes from [discord docs](
https://discord.com/developers/docs/resources/user#update-user-application-role-connection-json-params)